### ISSUE TITLE
W1-Submission?

### ISSUE THREAD

**Issue by codepathreview** [ani6gup/MovieMadness#1]
Have you completed your app and ready for submission?

**Comment by ani6gup** [comment]
Yes! My app is ready to be reviewed, week1 and week2 features have been implemented.

I apologize for the lack of the gitignore creating unnecessary files to be uploaded. 

Thank you for reviewing my app, /cc @codepathreview

**Comment by codepathreview** [comment]
Hi Anisha, thank for your submission. Please remember to create the issue and cc us so that we get notified immediately. 

However, I can't build your project as the main.storyboard is missing. Can you check and push your code again?

![Uploading Screen Shot 2016-01-31 at 10.37.17 PM.png…]()

**Comment by ani6gup** [comment]
I apologize for the missing element, I realized that at one point I accidentally deleted the storyboard and it was still linked to the trash. I fixed it and now it should be uploaded! I will definitely create my own issue next time as well! Thank you!

@codepathreview

**Comment by codepathreview** [comment]
:+1: Nice work! The point of this homework was to get a chance to implement a TableView (one of the most common views in iOS) and to work with real data over the network (in this case from the Movies Database API).

A key part of these projects is that you add additional features and tweak the UI / UX because that will provide the most learning opportunities. We encourage you to complete the projects early each week with the required stories and then spend time adding your own UI elements and experimenting with optional extensions that will improve the user experience.

Some notes:
- I see you also implement Network error, why don't you check that? Or you can't make the code run to that error? 1 tip, because iOS uses the cache, so even you disconnect, but you go back to the same link, the API always returns (but actually it's returned from cache, not from network). To do that, you can disable the cache and see. `let request = NSURLRequest(URL: url!, cachePolicy: .ReloadIgnoringLocalCacheData, timeoutInterval: 100)`
- When I see the Network error, I can't do anything else. You should provide user a way to close the error, or make it disappear when network is back.
<img width="322" alt="screen shot 2016-02-01 at 5 44 26 pm" src="https://cloud.githubusercontent.com/assets/7917093/12715326/54b5ac96-c90c-11e5-9554-bed94d557dde.png">
- It's a nice trick to load the low res image first before loading the high res. Good practice for UX.

We have a detailed [Project 1 Feedback Guide](http://courses.codepath.com/snippets/ios_university/feedback_guides/project_1_feedback.md) which covers the best practices for implementing this assignment. Read through the feedback guide point-by-point to determine ways you might be able to improve your submission. You should consider going back and implementing these improvements as well. Keep in mind that one of the most important parts of iOS development is learning the correct patterns and conventions.

If you have any particular questions about the assignment or the feedback, feel free to reply here or email us at <universitysupport@codepath.com>.

PS: Next time please create the issue following the naming convention: W[week#]-Submission
